### ISSUE TITLE
MySQL - statistics column not found

### ISSUE THREAD

**Issue by ringo360** [SphinxHD/MySQL-Sync#3]
An error occurred during data synchronization.
The statistics column does not seem to exist.
I would like to create the column manually as a stopgap measure until this issue is resolved,How can I create it?
> (Sorry, this is a translation from Japanese to English, so there may be some parts that are difficult to convey :( )

Server: `git-Purpur-1858 (MC: 1.19.2)`
MySQL-Sync version: `1.4.1(Latest)`
Log:
```
[11:34:25 WARN]: java.sql.SQLException: Column 'statistics' not found.
[11:34:25 WARN]:        at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
[11:34:25 WARN]:        at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
[11:34:25 WARN]:        at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
[11:34:25 WARN]:        at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
[11:34:25 WARN]:        at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:584)
[11:34:25 WARN]:        at com.mysql.cj.jdbc.result.ResultSetImpl.getString(ResultSetImpl.java:896)
[11:34:25 WARN]:        at MySQL-Sync-1.4.1.jar//hd.sphinx.sync.mysql.ManageMySQLData.loadPlayer(ManageMySQLData.java:147)
[11:34:25 WARN]:        at MySQL-Sync-1.4.1.jar//hd.sphinx.sync.MainManageData.loadPlayer(MainManageData.java:103)
[11:34:25 WARN]:        at MySQL-Sync-1.4.1.jar//hd.sphinx.sync.listener.JoinListener$1.run(JoinListener.java:59)
[11:34:25 WARN]:        at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:101)
[11:34:25 WARN]:        at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
[11:34:25 WARN]:        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1500)
[11:34:25 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:486)
[11:34:25 WARN]:        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1424)
[11:34:25 WARN]:        at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1194)
[11:34:25 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:310)
[11:34:25 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
[11:34:25 WARN]: [MySQL-Sync] Something went wrong with loading a Player!
```

**Comment by SphinxHD** [comment]
Hello there,
sorry for the late response, I was on a run the last two days.

You can create it manually using following MySQL Syntax:
`ALTER TABLE sync.playerdata ADD statistics LONGTEXT NULL;`

I will investigate into this error as soon as possible.
Can you send me some more informations, such as the Versions you used before if you used any before, etc.?

~ Leo

**Comment by ringo360** [comment]
This problem occurred when updating from 1.2.1 to 1.4.0/1.4.1.

**Comment by SphinxHD** [comment]
Hello,

I forgot to close this Issue, it should be resolved with 1.4.2.

~ Leo
